app/operations/candidates/upload_file.rb

```diff
@@ -14,6 +14,8 @@ def call
       add_event(attachment:, file:, actor_account:)
     end
 
+    update_profile_from_cv(candidate:, file:, actor_account:) if cv
```

Comment:
please use `case` for this operation calling
just display that we do nothing

Reply:
case here, or inside the method?

Reply:
hmmm, i'd prefer here. but up to you